confd/production.tmpl

```diff
@@ -16,6 +16,12 @@
       "database": "{{ getv "/configuration/db/database" "export-request" }}",
       "synchronize": {{ getv "/configuration/db/synchronize" "true" }},
       "logging": {{ getv "/configuration/db/logging" "false" }},
+      "ssl": {
+         "rejectUnauthorized": {{ getv "/configuration/db/ssl/reject_unauthorized" "null" }},
```

Comment:
Add to Helm charts

Reply:
https://github.com/MapColonies/raster-helm/pull/6